src/OpenSage.Game/Data/Rep/ReplayChunk.cs

```diff
@@ -69,6 +69,21 @@ internal static ReplayChunk Parse(BinaryReader reader)
                             order.AddScreenRectangleArgument(reader.ReadRectangle());
                             break;
 
+
+                            
+                        case OrderArgumentType.Unknown4:
+                            //in order to align bytes in a random replay, we needed to read 4. has to do with DrawBoxSelection
+                            order.AddIntegerArgument(reader.ReadInt32());
+                            //skip silently
+                            break;
+
+                            /*
+                        case OrderArgumentType.Unknown10:
```

Comment:
added a comment as to why it's there instead

Reply:
I disagree. If I remove it, there's no suggestion that there might be some work around this subject in the repository. So, unless people will trace all repository history in the event they encounter something new, this information will be missed. The alternative would be to include the code, but it would be a bit too "voodoo" for my tastes, that's why I commented it out, as a hint for somebody dealing with a related problem in the future :)